common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1601,3 +1601,54 @@ def can_toggle_course_pacing(self):
           bool: False if the course has already started, True otherwise.
         """
         return datetime.now(UTC()) <= self.start
+
+
+class CourseSummary(object):
+    """
+    A lightweight course summary class, which constructs split/mongo course summary without loading
+    the course. It is used at cms for listing courses to global staff user.
+    """
+    default_display_name = "Empty"
+    course_info_fields = ['display_name', 'display_coursenumber', 'display_organization']
+
+    def __init__(self, course_locator, course_summary):
+        """
+        Initialize and construct course summary
+        """
+        self.display_coursenumber = None
+        self.display_organization = None
+
+        self.course_summary = course_summary
```

Comment:
kind of confusing to call this variable `course_summary`. What about `course_metadata` or something else so we can better distinguish between the data that goes into `CourseSummary` objects and the objects themselves